.travis.yml

```diff
@@ -1,7 +1,8 @@
 language: ruby
 script: "bundle exec rake validate lint spec"
 rvm:
-  - 2.1.9
+  - 2.5.3
```

Comment:
I think this should match the respective ruby versions in the puppet-agent package, so something like this should do it:
```yaml
  include:
    - rvm: 2.1.9
      env: PUPPET_VERSION=4.10.12
    - rvm: 2.4.1
      env: PUPPET_VERSION=5.5.10
    - rvm: 2.5.1
      env: PUPPET_VERSION=6.2.0
```